packages/bot-skeleton/src/scratch/hooks/block_svg.js

```diff
@@ -269,7 +269,19 @@ Blockly.BlockSvg.prototype.setCollapsed = function (collapsed) {
             this.setCollapsed(false)
         );
 
-        this.appendDummyInput(COLLAPSED_INPUT_NAME).appendField(field_label).appendField(field_expand_icon).init();
+        if (this.type === 'procedures_defreturn' || this.type === 'procedures_defnoreturn') {
+            const funcLocale = this.getInput('').fieldRow[0].text_;
```

Comment:
as we apply this change only for function. and we know the first word is "function" it's better to use localize('function')